modules/apps/app-builder/app-builder-test/src/testFunctional/paths/AppBuilder.path

```diff
@@ -135,6 +135,42 @@
 	<td></td>
 </tr>
 
+<tr>
+	<td>DEPLOY_APP_STATUS</td>
+	<td>//span[contains(text(),'Deployed')]</td>
+	<td></td>
+</tr>
+
+<tr>
+	<td>UNDEPLOY_APP_BUTTON</td>
+	<td>//button[@class='btn btn-primary']</td>
+	<td></td>
+</tr>
+
+<tr>
+	<td>PRODUCT_MENU_DEPLOYED</td>
+<td>//span[@class='control-menu-level-1-heading text-truncate']</td>
+	<td></td>
+</tr>
+
+<tr>
+	<td>STANDALONE_DEPLOYED</td>
+<td>//h1[contains(@class,'app-builder-standalone-name')]</td>
+	<td></td>
+</tr>
+
+<tr>
+	<td>PORTLET_UNDEPLOYED</td>
+<td>//div[@class='alert alert-info']</td>
+	<td></td>
+</tr>
+
+<tr>
+	<td>WIDGET_DEPLOYED</td>
+<td>//h1[@class='taglib-empty-result-message-title']</td>
```

Comment:
Hi @nkkwon , as you did on PRODUCT_MENU_DEPLOYED validating the name of the app, could you please do the same here?

Maybe this path could be helpful: "//*[contains(@class,'portlet-title-text')]"